documentation/modules/ROOT/pages/connectors/index.adoc

```diff
@@ -16,4 +16,5 @@ We currently have the following connectors:
 [NOTE]
 ====
 An incubating connector is one that has been released for preview purposes and is subject to changes that may not always be backward compatible.
+For all the connectors that allow to track transactions, please note that this information is limited to transactions created after Debezium was installed and configured.
```

Comment:
Thanks, @lucamaf!

@roldanbob, Luca suggested to add this clarification, originating from some discussion in the support context. I think we should put this to where we talk about transaction metadata in the different connector docs. If you agree, can you perhaps pick this up and rework this PR accordingly? Thx!

Reply:
@gunnarmorling These appear to be two separate, unrelated notes, is that correct? That is, the transaction tracking limitations are independent of whether a connector is in an incubating state.

Reply:
A quick look tells me that we discuss transaction metadata for the following connectors:

- Db2,
- MongoDB
- Oracle, 
- PG
- SQL Server

Let me know if that's accurate.